clickhouse-native-jdbc/src/main/java/com/github/housepower/data/ColumnWriterBufferFactory.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.github.housepower.data;
+
+import java.util.concurrent.ConcurrentLinkedDeque;
+
+/**
+ * Factory of column writer that can recycle {@link ColumnWriterBuffer}.
+ * 
+ * <p>
+ * The factory is thread-safe and can be used by multiple threads or connections.
+ * </p>
+ *
+ */
+public class ColumnWriterBufferFactory {
+
+    private final ConcurrentLinkedDeque<ColumnWriterBuffer> stack = new ConcurrentLinkedDeque<>();
```

Comment:
I think it's better to export a method named `clear` to clear all buffers manually.